mmv1/products/firebaseappcheck/RecaptchaEnterpriseConfig.yaml

```diff
@@ -0,0 +1,81 @@
+# Copyright 2024 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+!ruby/object:Api::Resource
+name: "RecaptchaEnterpriseConfig"
+base_url: projects/{{project}}/apps/{{app_id}}/recaptchaEnterpriseConfig
+self_link: projects/{{project}}/apps/{{app_id}}/recaptchaEnterpriseConfig
+create_url: projects/{{project}}/apps/{{app_id}}/recaptchaEnterpriseConfig?updateMask=tokenTtl,siteKey
+create_verb: :PATCH
+update_verb: :PATCH
+update_mask: true
+skip_delete: true
```

Comment:
Any particular reason to set this field to true?

Reply:
The underlying API doesn't have a `DELETE` method. https://firebase.google.com/docs/reference/appcheck/rest/v1/projects.apps.recaptchaV3Config

I've considered an empty patch, but after discussing with the App Check experts, we've decided to make deletion a no-op. This is because an empty patch doesn't do what the user think it does. It doesn't make App Check "forget about" the app, but rather makes the config invalid (blank) which will break their app if still in use.

Reply:
Just want to make sure, if the API doesn't have a delete method, how shall this resource be deleted at the end of the test?

Reply:
Sounds good! Good to know that this resource won't cause any leak.